src/pyggdrasil/visualize/tree.py

```diff
@@ -67,7 +67,7 @@ def plot(
         dot_string += line
     # convert to networkX graph
     graphs = pydot.graph_from_dot_data(dot_string)
-    graph = graphs[0]
+    graph = graphs[0]  # type: ignore
```

Comment:
Took the liberty to suppress a false positive type warning here, that was causing your builts to fail.

Reply:
Wow, thanks a lot! :slightly_smiling_face: